cepheus:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  # Position Controllers ---------------------------------------
  left_shoulder_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_shoulder
    pid: {p: 10.0, i: 0.1, d: 1.0}
    # pid: {p: 1.0, i: 0.1, d: 1.0}
  # Position Controllers ---------------------------------------
  left_elbow_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_elbow
    pid: {p: 10.0, i: 0.1, d: 1.0}
    # pid: {p: 1.0, i: 0.1, d: 1.0}
  # Velocity Controllers ---------------------------------------
  reaction_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: reaction_wheel_joint
    pid: {p: 1.0, i: 0.1, d: 0.001}

  # new controllers

  # Position Controllers ---------------------------------------
  p1_position_controller:
    type: effort_controllers/JointPositionController
    joint: p1_joint
    pid: {p: 50.0, i: 0.1, d: 30.0}
  
  # Position Controllers ---------------------------------------
  p2_position_controller:
    type: effort_controllers/JointPositionController
    joint: p2_joint
    pid: {p: 50.0, i: 0.1, d: 30.0}
